src/link/style.scss

```diff
@@ -37,13 +41,51 @@
 			}
 		}
 
+		&-language {
```

Comment:
I think the box need more space on the left, it's now so close to the line border;

| ![image](https://user-images.githubusercontent.com/2560096/137970242-31fc64bd-eb61-4c1c-87db-bf558c36eb32.png) | ![image](https://user-images.githubusercontent.com/2560096/137970174-a52d89b5-8f8f-4d2b-93db-ae117125999a.png) |
| --- | --- |

another styling icon, it looks offset now (same to mobile header)

| ![image](https://user-images.githubusercontent.com/2560096/137970405-85a17085-8dc4-4b12-b37d-680d3d0bc65a.png) | ![image](https://user-images.githubusercontent.com/2560096/137970487-353f150e-bf09-4d25-8a0a-48196a46770d.png) |
| --- | --- |